.github/workflows/wrappers.yml

```diff
@@ -310,63 +326,18 @@ jobs:
       id: check-cache
       uses: nirinchev/cache@d7c96a77c26ab70dd32b202c885cb4b34d95d8a8
       with:
-        path: ./wrappers/cmake/**/realm-wrappers.framework
+        path: ./wrappers/build/**
         key: wrappers-ios-${{ matrix.arch }}-Release-${{hashFiles('./wrappers/**')}}
     - name: Build wrappers
-      run: pwsh ./wrappers/build-ios.ps1 ${{ matrix.arch }} -SkipXCFramework -Configuration Release
+      run: pwsh ./wrappers/build-ios.ps1 ${{ matrix.arch }} -Configuration Release
       if: steps.check-cache.outputs.cache-hit != 'true'
     - name: Store artifacts for wrappers-ios-${{ matrix.arch }}
       uses: actions/upload-artifact@v2
       with:
         name: wrappers-ios-${{ matrix.arch }}
-        path: ${{ github.workspace }}/wrappers/cmake/**/realm-wrappers.framework
-        retention-days: 1
-  ios:
-    runs-on: macos-latest
-    name: iOS XCframework
```

Comment:
Are we no longer producing an xcframework?

Reply:
No, we ship the device framework under `runtimes/ios` and the simulator framework under `runtimes/iossimulator`.